### ISSUE TITLE
[Requests] Global Hotkey to show/hide the app window. Also a way to have multiple note pages that can be switched between with a hotkey

### ISSUE THREAD

**Issue by MostHated** [Kilian/fromscratch#115]
Hey there,
    I really like the simplicity of the app, but I feel the one thing it is missing is ultra-quick accessibility. My thoughts are, if I have an idea or information I need to jot down quickly, a simple app like this is great, but unfortunately, if I have to go looking for an icon to click on a bar or in the start menu, I may as well just open Sublime Text or VSCode, etc. What would truly make this app standout and shine is the ability for it to be right there when you need it as quickly as possible. 

Being able to just hit a key/hotkey combo on the keyboard and then it is right there, window focused, ready to start typing. I type my note I needed to remember, hit the key combo again and now the window is gone and I can continue on with what I was doing just as quickly. That is exactly the type of note taking app I have been looking for, for a long time, but for some reason just can't find.

That said, I often have Sublime text open with about 10 different tabs that I use for different types of notes/scratch pads, so if I could use this instead and be able to switch between a few different pages, it would be great!

While writing this, though, I just realized that if I hit f11 and make it fullscreen size, there is no way to bring the size back down at all, so now I am sort of stuck as I hit f11 on my main monitor, which is a 34in wide, so it's way too large for my other monitors, and hitting f11 again doesn't do anything, and I can't manually resize it.

Thanks,
-MH

**Comment by nzetsu** [comment]
***Using debian 10 buster***

Hi MostHated, 

> I really like the simplicity of the app, but I feel the one thing it is missing is ultra-quick accessibility. My thoughts are, if I have an idea or information I need to jot down quickly, a simple app like this is great, but unfortunately, if I have to go looking for an icon to click on a bar or in the start menu, I may as well just open Sublime Text or VSCode, etc. What would truly make this app standout and shine is the ability for it to be right there when you need it as quickly as possible.

> Being able to just hit a key/hotkey combo on the keyboard and then it is right there, window focused, ready to start typing. I type my note I needed to remember, hit the key combo again and now the window is gone and I can continue on with what I was doing just as quickly. That is exactly the type of note taking app I have been looking for, for a long time, but for some reason just can't find.

If you haven't already found a solution or I must better say workaround for your convenience, you could try with some general hotkey combinations. For me I added the FromScratch App to my Favorites and I can open it quickly by typing **cmd/super+position_of_the_App** e.g. On my main desktop FromScratch is at position 3 of my apps favorites and by typing **cmd/super+3** the App is under 1s there and I can start editing. Also, I generally prefer to use **cmd/super+h** to hide the app (instead of having a minimize button on the window Titlebar). Additionally i use **cmd/super+up**, **cmd/super+left**, ****cmd/super+right**** or **cmd/super+down** to put the App-window where i want it to be. 

> That said, I often have Sublime text open with about 10 different tabs that I use for different types of notes/scratch pads, so if I could use this instead and be able to switch between a few different pages, it would be great!**


 For this one, I simply make use of the portable mode [see here](https://github.com/Kilian/fromscratch#download) and can open my notes every time i want to.

> While writing this, though, I just realized that if I hit f11 and make it fullscreen size, there is no way to bring the size back down at all, so now I am sort of stuck as I hit f11 on my main monitor, which is a 34in wide, so it's way too large for my other monitors, and hitting f11 again doesn't do anything, and I can't manually resize it.

See the other paragraph with all the use of simple shortcuts to play around whit windows. As you might already know you can modify all the key combinations to suit your needs. 

Hopefully, I could help here. If you know better ways to accomplish all the tasks or I wrote some none sense above, please let me/us know. 

bye-bye
